.github/workflows/on-prerelease.yml

```diff
@@ -0,0 +1,223 @@
+
+jobs:
+  pack:
+    name: Build and Pack
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: pwsh
+    outputs:
+      hash-code: ${{ steps.hash-code.outputs.hash-code }}
+      appcommon-v: ${{ steps.versions.outputs.appcommon-v }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          fetch-depth: 0
+
+      - name: Set Version Numbers
+        id: versions
+        shell: pwsh
+        run: |
+          Import-Module ./github-builder.psm1
+          Get-VersionNumber
+
+      - name: Create NuGet Package
+        shell: pwsh
+        run: |
+          $packageVersion = "${{ steps.versions.outputs.appcommon-semver }}"
+
+          Import-Module ./github-builder.psm1
+
+          Invoke-DotnetPack $packageVersion
+
+      - name: Generate hash for NuGet package
+        id: hash-code
+        shell: pwsh
+        run: |
+          "hash-code=$(sha256sum *.nupkg | base64 -w0)" | Out-File -FilePath $env:GITHUB_OUTPUT -Append
+
+      - name: Upload Packages as Artifacts
+        if: success()
+        uses: actions/upload-artifact@26f96dfa697d77e81fd5907df203aa23a56210a8 # v4.3.0
+        with:
+          name: "${{ env.PACKAGE_NAME }}-NuGet"
+          path: ${{ github.workspace }}/*.nupkg
+          if-no-files-found: error
+          retention-days: 30
+
+  sbom-create:
+    name: Create SBOM
+    runs-on: ubuntu-latest
+    needs: pack
+    permissions:
+      actions: read
+      contents: write
+    outputs:
+      sbom-hash-code: ${{ steps.sbom-hash-code.outputs.sbom-hash-code }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          fetch-depth: 0
+
+      - name: Get Artifact
+        uses: actions/download-artifact@6b208ae046db98c579e8a3aa621ab581ff575935 #v4.1.1
+        with:
+          name: ${{ env.PACKAGE_NAME }}-NuGet
+
+      - name: Generate Software Bill of Materials (SBOM)
+        shell: pwsh
+        run: |
+          $packageName = "${{ env.PACKAGE_NAME }}"
+          $version = "${{ needs.pack.outputs.appcommon-v }}"
+
+          $url = "${{ env.SBOM_TOOL_URL }}"
+          "${{ env.SBOM_TOOL_HASH }}" | Out-File hash-value
+          $out = "$($env:RUNNER_TEMP)/sbom-tool-linux-x64"
+          Invoke-RestMethod -Uri $url -OutFile $out
+
+          pushd $($env:RUNNER_TEMP)
+          md5sum --check hash-value
+          popd
+
+          chmod +x $out
+
+          (Resolve-Path ("./$($packageName).$($version).nupkg")).Path | Out-File -FilePath buildfilelist.txt
+
+          &$out generate `
+              -b ./ `
+              -bl ./buildfilelist.txt `
+              -bc "./" `
+              -pn "$packageName" `
+              -pv $version `
+              -nsb https://ed-fi.org `
+              -m ./ `
+              -ps "Ed-Fi Alliance"
+
+          cp _manifest/spdx_2.2/manifest.spdx.* .
```

Comment:
This is another adjustment that I had to make. I was running into strange errors when the artifact was a directory of files. I was able to resolve by moving the needed files out of the sub-directory.